src/RevCore.cc

```diff
@@ -17,27 +17,26 @@ namespace SST::RevCPU {
 using MemSegment = RevMem::MemSegment;
 
 RevCore::RevCore(
-  unsigned                  Id,
-  RevOpts*                  Opts,
-  unsigned                  NumHarts,
-  RevMem*                   Mem,
-  RevLoader*                Loader,
+  unsigned                  id,
+  RevOpts*                  opts,
+  unsigned                  numHarts,
+  RevMem*                   mem,
+  RevLoader*                loader,
   std::function<uint32_t()> GetNewTID,
-  SST::Output*              Output
+  SST::Output*              output
 )
-  : Halted( false ), Stalled( false ), SingleStep( false ), CrackFault( false ), ALUFault( false ), fault_width( 0 ), id( Id ),
-    HartToDecodeID( 0 ), HartToExecID( 0 ), numHarts( NumHarts ), opts( Opts ), mem( Mem ), coProc( nullptr ), loader( Loader ),
-    GetNewThreadID( std::move( GetNewTID ) ), output( Output ), feature( nullptr ), sfetch( nullptr ), Tracer( nullptr ) {
+  : id( id ), numHarts( numHarts ), opts( opts ), mem( mem ), loader( loader ), GetNewThreadID( std::move( GetNewTID ) ),
+    output( output ) {
```

Comment:
Most of the members are initialized to `0` with default initializers now, so there is no need to crowd the constructor initialization list.

Reply:
See previous comment about capitalization of constructor parameters being unnecessary to distinguish them from members.